net/Kconfig

```diff
@@ -117,6 +117,15 @@ config NET_GUARDSIZE
 		packet size will be chopped down to the size indicated in the TCP
 		header.
 
+config NET_LL_GUARDSIZE
+	int "Data Link Layer(L2) Guard size of Network buffer(IOB)"
+	default 14 if NET_ETHERNET
```

Comment:
why not use zero too

Reply:
NET_LL_GUARDSIZE cannot be smaller than the current maximum l2 header length